### DISCUSSION TITLE
check AdMob SDK version

### DISCUSSION THREAD

**Discussion by lucasaugustin** [invertase/react-native-firebase#4369] [Q&A] [answered]
Hi guys.

I need to check what version of AdMob and GMA SDK this module is using, but i don't find it in any documentations or file.
Any ideia where to find it?

**Comment by mikehardy** [comment, marked as answer]
It's a transitive dependency. For Android you want `cd android && ./gradlew :app:dependencies` and on iOS it's `cd ios && cat Podfile.lock`
